README.md

```diff
@@ -314,9 +314,6 @@ For more information on limitations and recommendations, please refer to [IAM da
 For more information, please refer to [Creating a database account using IAM authentication](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.DBAccounts.html#UsingWithRDS.IAMDBAuth.DBAccounts.PostgreSQL).
 
 #### Using AWS IAM Database Authentication
```

Comment:
Sorry wasn't being clear. We still need the part explaining how to enable it, so the setting the property `AUTHENTICATION_PLUGIN_CLASS_NAME` as `AwsIamAuthenticationPlugin` part is still needed.